Code/Framework/AzFramework/AzFramework/Archive/ZipDirStructures.cpp

```diff
@@ -187,8 +187,8 @@ namespace AZ::IO::ZipDir::ZipDirStructuresInternal
 
         // If src/dst overlap (in place decompress), then inflate in chunks, copying src locally to ensure
         // pointers don't foul each other.
-        bool bIndependantBlocks = ((pInput + nInputLen) <= pOutput) || (pInput >= (pOutput + nOutputLen));
-        if (bIndependantBlocks)
+        bool bIndependentBlocks = ((pInput + nInputLen) <= pOutput) || (pInput >= (pOutput + nOutputLen));
+        if (bIndependentBlocks)
```

Comment:
nit: Now that you're changing the name it's a good opportunity to remove the type prefix as that's naming convention is not part of the standard.

Reply:
Ok